lib/elixir/lib/module/locals_tracker.ex

```diff
@@ -107,7 +107,8 @@ defmodule Module.LocalsTracker do
       for {pair, _, meta, _} <- all_defined,
           {local, position, macro_dispatch?} <- out_neighbours(bag, {:local, pair}),
```

Comment:
Nitpick: unpack the local here:

```suggestion
          {{name, _arity} = local, position, macro_dispatch?} <- out_neighbours(bag, {:local, pair}),
```